### ISSUE TITLE
Unable to exit out of text editor using keyboard alone

### ISSUE THREAD

**Issue by IshavSohal** [ramp4-pcar4/storylines-editor#551]
## Description

When focus is within a text editor, you are unable to exit it using the keyboard alone

## Steps to reproduce

1. Open a product with a text panel, and open that panel
2. Tab to the text editor
3. Try to exit it using keyboard alone

## Expected behavior

There should be some combination of keys that allows keyboard users to exit the text editor

## Actual behavior

No such combination of keys (that I could find)
